doc/topics/releases/2015.5.4.rst

```diff
@@ -11,3 +11,7 @@ Changes:
 
 - When querying for VMs in ``ditigal_ocean_v2.py``, the number of VMs to include in a page was changed from 20
```

Comment:
@jbouse that typo wasn't caused by this Pull Request and it already existed before. Feel free to submit a PR to fix it (I'm assuming it should have been already fixed by now in develop branch)